.github/CODEOWNERS

```diff
@@ -26,7 +26,7 @@
 /docs/source-app/expertise_levels           @williamfalcon @Felonious-Spellfire @RobertLaurella
 
 # Packages
-/src/pytorch_lightning                      @carmocca @justusschock
+/src/pytorch_lightning                      @borda @awaelchli @carmocca @justusschock @rohitgr7
```

Comment:
Can I piggyback on this and become a codeowner too?

Reply:
```suggestion
/src/pytorch_lightning                      @borda @awaelchli @carmocca @justusschock @rohitgr7 @otaj
```